.mockery.yaml

```
@@ -0,0 +1,25 @@
+with-expecter: true
+filename: "{{.InterfaceName | lower }}.generated.go"
```

```suggestion
filename: "{{.InterfaceName | lower }}_mock_test.go"
```

How about just defining the file name here globally?

---

Cannot do that if we move mocks to "mocks" dir. See below.